device/isis/prefixattributes.yaml

```diff
@@ -0,0 +1,27 @@
+components:
+  schemas:
+    Isis.PrefixAttributes:
+      description: >-
+        This contains the properties of ISIS Prefix attributes for 
+        the extended IPv4 and IPv6 reachability.
+      type: object
+      properties:
+        x_flag:
+          description: |-
+            External Prefix Flag (Bit 0)
+          type: boolean
+          default: false
+        r_flag:
+          description: |-
+            Re-advertisement Flag (Bit 1)
+          type: boolean
+          default: false
+        n_flag:
+          description: |-
+            Node Flag (Bit 2)
+          type: boolean
+          default: false
+
```

Comment:
Can we add the remaining 5 TBD flags here? These flags are getting defined by various IETF drafts, I think it will be good if we can expose these flags even at this initial stage.

Reply:
These more flags can be added in future too.